examples/vols/ex.yml

```diff
@@ -0,0 +1,34 @@
+version: 0.1-dev
+
+services:
+- name: db
+  containers:
+  - image: mysql
+    mounts:
+    - volumeName: db
+      mountPath: /var/lib/mysql
+
+- name: backup
+  containers:
+  - image: backup
+    mounts:
+    - volumeName: db
+      mountPath: /app/store
+      volumeSubPath: foo/bar
```

Comment:
> Would it be reasonable to include support for paths like $HOME in the spec?

That is interesting idea. We won't add it in this initial implementation, but definitely will keep it in mind for future improvements. Will have to do some research on how this can be done in Kubernetes. (any pointers?)